app/accounts/mutations/visitors.py

```diff
@@ -1,7 +1,7 @@
 import graphene
```

Comment:
Aqui por exemplo, é necessário colocar a [docstring de módulo](https://stackoverflow.com/questions/2557110/what-to-put-in-a-python-module-docstring#2557196)